bigquery/data-transfer/api/QuickStartTest/QuickStartTest.cs

```diff
@@ -0,0 +1,31 @@
+/*
+ * Copyright (c) 2018 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+using Xunit;
+
+namespace GoogleCloudSamples
+{
+    public class QuickStartTest
+    {
+        [Fact]
+        public void TestMain()
+        {
+            // Main() will throw an exception if something fails.
+            QuickStart.Main(new string[] { });
```

Comment:
Typically, you wouldn't write tests this way for real application. I'd suggest you extract main's method into a separate method, then test that method. Main method will call the same method too. It may look like we add more unnecessary work here, but it will mimic real world better.

Reply:
This file is exactly copied from https://github.com/GoogleCloudPlatform/dotnet-docs-samples/blob/master/datastore/api/QuickStartTest/QuickStartTest.cs